### PR TITLE
Slightly changed scripts to enable Chaos Mesh Dashboard.

### PR DESCRIPTION
**Pull request by ClSlaid**
### What problem does this PR solve?

Corrected script in user guide.
Some scripts doesn't fit previous steps well.
### What is changed and how does it work?
A script about enabling Chaos Mesh Dashboard was changed.
### Checklist

No file removed.
Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
Corrected single script in user guide.
```

### CODE REVIEW

**Comment by fewdan** [comment]
How about we communicate in slack channel?

**Comment by ClSlaid** [comment]
> How about we communicate in slack channel?

I prefer telegram instead, I'm not familiar with slack...

**Comment by fewdan** [comment]
try [this](https://github.com/chaos-mesh/chaos-mesh/pull/829/checks?check_run_id=1007004770)

**Comment by ClSlaid** [comment]
I added a command line in document to make accessing to web-show application running on servers more comfortable.
```kubectl port-forward -n default svc/web-show 8081:8081```

File: website/docs/user_guides/run_chaos_experiment.md [hunk 1]
```diff
@@ -19,7 +19,8 @@ After executing the above command, you can access [`http://localhost:8081`](http
 > **Note:**
 >
 > If the web-show is deployed on the server, you need to use the host ip to access the application.
-
+> However by executing script below enables you to view web-show application by accessing [`http://localhost:8081`](http://localhost:8081). 
+> ```kubectl port-forward -n default svc/web-show 8081:8081```
```
Comment (ClSlaid) [inline website/docs/user_guides/run_chaos_experiment.md:23]: Port-forward <targetIP:8081> to <localhost:8081>

**Comment by zhouqiang-cl** [comment]
@ClSlaid how about this pr now

**Comment by ClSlaid** [comment]
> 
> 
> @ClSlaid how about this pr now
That's wierd and embarrasing.
I tried hard and nothing worked at all.
I'll look for help from my classmate later to solve it.

**Pull request closed without merging**